lms/envs/common.py

```diff
@@ -848,6 +848,15 @@
 ##### Feedback submission mechanism #####
 FEEDBACK_SUBMISSION_EMAIL = None
 
+#### Help Modal Links ####
+if FEATURES.get('ENABLE_FEEDBACK_SUBMISSION'):
```

Comment:
I had it this way since the template doesn't try to access HELP_MODAL_LINKS unless ENABLE_FEEDBACK_SUBMISSION is enabled. See: https://github.com/Stanford-Online/edx-platform/blob/master/lms/templates/help_modal.html#L13